### ISSUE TITLE
Unify the units as much as possible

### ISSUE THREAD

**Issue by RobinJadoul** [be-oi/beoi-training#3]
Adapt the slides, such that everything is nice and homogenous.

Some changes that may be needed:
[ ] The color scheme of the slides (let's just go with white? maybe a custom style :smile:)
[ ] I wouldn't keep the dates on the slides; maybe remove the names as well
[ ] Make sure the README is there and easily available
[ ] Make all big-oh's fancy (`\mathcal{O}`) :)
[ ] Get a consistent name for where source code for that unit is stored (instead of having 'code', 'src', 'lst', ...)

Some things need deciding what to do with them, so say what you think...

If there are more points that need unifying, say so here as well

**Comment by mcouplet** [comment]
`\newcommand{\bigoh}[1]{\ensuremath{\mathcal{O}\left(#1\right)}}` :)

**Comment by mcouplet** [comment]
Let's vote for the name of the source code location :)
:+1: `code`
:heart: `src`
:tada:  `lst`

**Comment by nicoekkart** [comment]
It should be code because that's the clearest for the students searching for code.

**Comment by RobinJadoul** [comment]
for the big-Oh newcommand, a custom latex style (which can still be modified later on, if we want/need more customisation) might be useful

**Comment by RobinJadoul** [comment]
A deadline for the decision on the source code location?

**Comment by mcouplet** [comment]
Well just make it `src` :)

**Comment by nicoekkart** [comment]
NO! We'll vote the closing in one year. I still have to make the 'code' propaganda flyers

**Comment by RobinJadoul** [comment]
Vote closed :smile:
